RAT/utils/misc.py

```diff
@@ -0,0 +1,29 @@
+"""Miscellaneous utility functions for RAT."""
+import importlib.util
+import os
+import pathlib
+from typing import Callable, Union
+
+
+def get_python_handle(file_name: str, function_name: str, path: Union[str, pathlib.Path] = "") -> Callable:
```

Comment:
Please move this into inputs.py where it is used, I would prefer it wasn't a one use function but if it necessary for testing at least place it in the only file that uses it.